app/views/declaration/summary/parties_section.scala.html

```diff
@@ -98,26 +98,47 @@ <h2 class="govuk-heading-m">Parties</h2>
             isEoriDefault = false
         )).getOrElse(Seq.empty)
 
+        ++ (Seq.empty :+ declarationData.parties.representativeDetails.map(_.representingOtherAgent).map(agent =>
```

Comment:
Instead of adding Seq.empty to element is enough to wrap element in Sequence

`Seq.empty :+ element` == `Seq(element)`